src/status/chat.nim

```diff
@@ -138,10 +138,16 @@ proc leave*(self: ChatModel, chatId: string) =
   # TODO: REMOVE MAILSERVER TOPIC
   # TODO: REMOVE HISTORY
   self.filters.del(chatId)
-  self.channels.del(chatId)  
+  self.channels.del(chatId)
   self.events.emit("channelLeft", ChatIdArg(chatId: chatId))
   self.events.emit("activeChannelChanged", ChatIdArg(chatId: ""))
 
+proc clearHistory*(self: ChatModel, chatId: string) =
+  discard status_chat.clearChatHistory(chatId)
```

Comment:
Clearing history always seems to return a `result: null` so there's nothing we can really do with the response.

We might wanna consider error handling though in case this API gets called with an invalid chat id